### PR TITLE
SDK 3.7.0 update

### PR DESCRIPTION
**Pull request by AmitaiB**
Cumulative update to the current SDK, 3.7.0

### CODE REVIEW

**Comment by AmitaiB** [comment]
PR is Draft, since 3.7.1 should be released any day now.

File: node_modules/global/process.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = require('process');
```
Comment (karimMourra) [inline node_modules/global/process.js:1]: @AmitaiB why are there node modules ?

**Comment by AmitaiB** [comment]
@henryhlee @karimMourra I am getting confused SMS people saying that the link to our demo is not up to date -- since all the work is on `develop`.

**Comment by karimMourra** [comment]
@AmitaiB i left you a comment in october

File: node_modules/global/process.js (hunk 1 above)
Comment (AmitaiB) [reply node_modules/global/process.js:1]: @karimMourra I do not know. I just assumed it was a Fastlane or other TE artifact. @mikesalvia ?

**Comment by AmitaiB** [comment]
@karimMourra Thanks, I missed that. I have now replied to that Oct comment. It would be great if we could resolve this soon.

File: node_modules/global/process.js (hunk 1 above)
Comment (karimMourra) [reply node_modules/global/process.js:1]: I don't see how, fastlane has nothing to do with this app, and those node modules are not in the framework directory so they were installed by you in your local project. 
@AmitaiB Please delete all those node modules, did you check your diff ? You have 241 file changes, the majority of which is unnecessary.

**Review by karimMourra — CHANGES_REQUESTED** [review]
You have 241 file changes, which is absurd and most of them are node_modules which have nothing to do with this project. Please delete all of them

File: node_modules/global/process.js (hunk 1 above)
Comment (karimMourra) [reply node_modules/global/process.js:1]: you even have a `package-lock.json` file in your diff. This is not a javascript project, you shouldn't have that

**Comment by AmitaiB** [comment]
@karimMourra I do not know where they came from -- all I did was update the framework, so it follows that those files are an artifact of that operation. Since you do not, I will close this PR, and open a fresh one.

In consultation with @henryhlee , we will try switching the demo app to cocoapods, since instances of CocoaPods-caused errors are rerlatively rare.

**Pull request closed without merging**